Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx

```diff
@@ -168,13 +197,25 @@ const MusicPlayer = () => {
         <View style={styles.container}>
             <Image
                 source={{
-                    uri: "https://cdn.pixabay.com/audio/2024/01/15/16-58-27-753_200x200.jpg",
+                    uri:
+                        currentSong.artwork === ""
+                            ? "https://cdn.pixabay.com/audio/2024/01/15/16-58-27-753_200x200.jpg"
+                            : `${currentSong.artwork}`,
                 }}
                 style={styles.albumCover}
             />
             <View style={styles.infoAndControls}>
-                <Text style={styles.songTitle}>Coverless book ( Lofi )</Text>
-                <Text style={styles.albumInfo}>AmbientAUDIOVISION</Text>
+                <Text style={styles.songTitle}>
```

Comment:
Display song info on music page

Reply:
Displaying song information demonstrates this feature better